src/main/java/noppes/npcs/scripted/entity/ScriptPlayer.java

```diff
@@ -407,6 +462,13 @@ public ScriptPixelmonPlayerData getPixelmonData(){
 			return null;
 		return new ScriptPixelmonPlayerData(player);
 	}
+	
+	public Vec3 getLookingAtBlock(int maxDistance) {
+		Vec3 posVec = Vec3.createVectorHelper(player.posX, 
+				player.posY+player.getEyeHeight(), player.posZ);
+		Vec3 lookVec = player.getLookVec();
+		return NpcAPI.Instance().getIWorld(player.dimension).rayCastBlockPos(posVec, lookVec, maxDistance);
+	}
```

Comment:
This should return an IBlock object instead.

Reply:
Will you be backporting `ScriptRayTrace`? If so this function is unnecessary.

Reply:
@1whohears Yes, it shouldn't be too hard!